website/blog/2024-12-22-wish-i-had-a-control-plane-for-my-renovation.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Why I wish I had a control plane for my renovation"
+description: "When I think back to my renovation, I realize how much smoother it would have been if I’d had a control plane for the entire process."
```

Comment:
```suggestion
description: "When I think back to my renovation, I realize how much smoother it would've been if I’d had a control plane for the entire process."
```